README.md

```diff
@@ -93,3 +93,40 @@ The configuration file is in `.json` format.
     }
 
 You have to restart the server for configuration changes to take effect.
+
+
+## Setup with v2 Dev-Environment
```

Comment:
With the changes in the dev-env, I think this is all done automatically now, and so doesn't need including here? Likewise with the OAuth app - the instructions for that should be to run `bin/run rails_v1 rake db:seed` in the dev environment?